src/main/java/uk/gov/laa/pfla/auth/service/services/ReportTrackingTableService.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.laa.pfla.auth.service.services;
+
+import org.springframework.stereotype.Service;
+import uk.gov.laa.pfla.auth.service.builders.ReportTrackingTableModelBuilder;
+import uk.gov.laa.pfla.auth.service.dao.ReportTrackingTableDao;
+import uk.gov.laa.pfla.auth.service.models.ReportTrackingTableModel;
+
+import java.time.LocalDateTime;
+
+@Service
+public class ReportTrackingTableService {
+
+    private ReportTrackingTableDao reportTrackingTableDao;
+
+    public ReportTrackingTableService(ReportTrackingTableDao reportTrackingTableDao) {
+        this.reportTrackingTableDao = reportTrackingTableDao;
+    }
+
+    public void updateReportTracking(int requestedId, LocalDateTime creationTime) {
+
+        ReportTrackingTableModel reportTrackingTableModel = new ReportTrackingTableModelBuilder()
```

Comment:
I assume hard coded values to be changed in next revision when SSO and DB connection is established